cli/src/main/java/org/aya/cli/parse/AyaProducer.java

```diff
@@ -173,7 +173,8 @@ public SeqView<UseHide.Name> useIdsComma(@NotNull GenericNode<?> node) {
       var asId = weakId(useAs.child(WEAK_ID)).data();
       var asAssoc = useAs.peekChild(ASSOC);
       var asBind = useAs.peekChild(BIND_BLOCK);
-      return new UseHide.Name(wholePos, name.data(), asId,
+      var qname = new QualifiedID(wholePos, name.data());    // TODO: Fix this
+      return new UseHide.Name(qname, asId,
```

Comment:
What's this?

Reply:
1. The source pos is incorrect
2. I don't update the grammar yet, so `using (Nat::zero as zero)` is still illegal.